pkgs/desktops/gnome/extensions/extensionOverrides.nix

```diff
@@ -166,4 +167,11 @@ super: lib.trivial.pipe super [
         --replace "GLib.build_filenamev([GLib.DIR_SEPARATOR_S, 'usr', 'share', 'touchegg', 'touchegg.conf'])" "'${touchegg}/share/touchegg/touchegg.conf'"
     '';
   }))
+
+  (patchExtension "custom-command-list@storageb.github.com" (old: {
+    postPatch = ''
+        substituteInPlace "extension.js" \
+          --replace "/bin/bash" "${bash}/bin/bash"
```

Comment:
`--replace-fail` is deprecated, please use `--replace`.
Also this would still break silently if they switch to e.g.  `/usr/bin/bash` so please include more context to the search pattern (e.g. quotes).